alvr/client_openxr/src/extra_extensions/face_tracking_pico.rs

```diff
@@ -0,0 +1,119 @@
+use openxr::sys::pfn::VoidFunction;
+use openxr::{self as xr, sys};
+use std::mem;
+
+const TRACKING_MODE_FACE_BIT: u64 = 0x00000008;
+const TRACKING_MODE_FACE_LIPSYNC: u64 = 0x00002000;
+const TRACKING_MODE_FACE_LIPSYNC_BLEND_SHAPES: u64 = 0x00000100;
+
+#[repr(C)]
+struct FaceTrackingDataPICO {
+    time: sys::Time,
+    blend_shape_weight: [f32; 72],
+    is_video_input_valid: [f32; 10],
+    laughing_probability: f32,
+    emotion_probability: [f32; 10],
+    reserved: [f32; 128],
+}
+
+type StartEyeTrackingPICO = unsafe extern "system" fn(sys::Session) -> sys::Result;
+
+type SetTrackingModePICO = unsafe extern "system" fn(sys::Session, u64) -> sys::Result;
+
+type GetFaceTrackingDataPICO = unsafe extern "system" fn(
+    sys::Session,
+    sys::Time,
+    i32,
+    *mut FaceTrackingDataPICO,
+) -> sys::Result;
+
+pub struct FaceTrackerPico {
+    session: xr::Session<xr::AnyGraphics>,
+    get_face_tracking_data: GetFaceTrackingDataPICO,
+}
+
+impl FaceTrackerPico {
+    pub fn new<G>(session: xr::Session<G>, visual: bool, audio: bool) -> xr::Result<Self> {
+        let mut tracking_flags = 0u64;
```

Comment:
There should be no need to specify the type in the literal (u64) but if you do, put a underscore in between: `0_u64`